src/PKSim.Core/Snapshots/Mappers/ExpressionProfileMapper.cs

```diff
@@ -154,8 +154,16 @@ private Localization retrieveLocalizationFrom(SnapshotExpressionProfile snapshot
 
       private bool isV9Format(SnapshotExpressionProfile snapshot)
       {
-         //Localization is only defined for protein and was added in v10
-         return snapshot.Localization == null && snapshot.TransportType == null;
+         //This was a protein
```

Comment:
That was the problem here. Trying to decide if this is a v9 format or not
In fact, we have a project version defined in the project file .I think this is what I should have used instead. However, we need to pass it top down which we do not have at the moment. A good enhancement that I might had before we release